src/main/java/creativei/controller/ReminderController.java

```diff
@@ -41,4 +42,10 @@ ResponseObject createReminder(@RequestBody String reminderString, HttpServletReq
             return ResponseObject.getResponse(ExceptionType.GENERAL_ERROR.getMessage(), ExceptionType.GENERAL_ERROR.getCode());
         }
     }
+
+    @GetMapping(value = "/reminder/getByDate",produces = "application/json")
+    public @ResponseBody
```

Comment:
Use json to wrap input params. Rename to toDate and fromDate

Reply:
done in commit 1a6703f

Reply:
How should i use json here?

Reply:
resolved in commit 3fe67e5